knowledge/textbook/culture/starwars/tvseries/theacolyte/qna.yaml

```diff
@@ -0,0 +1,34 @@
+task_description: 'Teach the model on Star Wars The Acolyte (TV series)'
+created_by: markus.van.kempen
```

Comment:
Please put this as your GitHub username.

Reply:
Update qna.yaml with version and github username